core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -48,4 +48,32 @@ void onError(
       long latencyNanos,
       DriverConfigProfile configProfile,
       Node node);
+
+  /**
+   * Invoked each time a request fails at the node level. Similar to onError but at a per node
+   * level.
+   *
+   * @param latencyNanos the overall execution time (from the {@link Session#execute(Request,
+   *     GenericType) session.execute} call until the error is propagated to the client).
+   * @param configProfile the configuration profile that this request was executed with.
+   * @param node the node that returned the error response, or {@code null} if the error occurred
+   */
+  void onNodeError(
+      Request request,
+      Throwable error,
+      long latencyNanos,
+      DriverConfigProfile configProfile,
+      Node node);
+
+  /**
+   * Invoked each time a request succeeds at the node level. Similar to on Success but at per Node
```

Comment:
`on Success` -> `onSuccess` and since referring to another method it would be useful to use `{@link #onSuccess(...)}` so you can click to it in javadoc.  (same for `onError` above).